client/components/dialog/dialog-base.jsx

```diff
@@ -16,6 +16,18 @@ var closeOnEsc = require( 'lib/mixins/close-on-esc' ),
 	Card = require( 'components/card' ),
 	trapFocus = require( 'lib/mixins/trap-focus' );
 
+// Subclass <Card> component to extend with "click outside" behavior.".
+var DialogBaseCard = withClickOutside( React.createClass( {
+
+	render: function() {
+		return <Card {...this.props} />
+	},
+
+	handleClickOutside: function( event ) {
+		this.props.onClickOutside( event );
```

Comment:
Yes. `DialogBaseCard` is essentially a private component class (not exposed by the `DialogBase` module). Its only purpose is to wrap the `Card` component with the `react-click-outside` mixin so that the `handleClickOutside` method can be passed through to the `DialogBase` component's `_onBackgroundClick` method.

Note: `react-click-outside` will not fail if the method is not present. It will only fire if it finds a `handleClickOutside` method on the wrapped component. However, the method name is hardcoded and so I've extended the `Card` component with the `handleClickOutside` method to be used only as a passthru.